src/components/Footer/Footer.js

```diff
@@ -1,9 +1,68 @@
+/* eslint-disable */
 import React from 'react';
+import { FaComment } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import './Footer.scss';
 
 const Footer = () => {
   return (
-    <div>
-      <h1>footer</h1>
+    <div className="footer">
+      <div className="footerInner">
```

Comment:
```suggestion
      <div className="container">
```
효율적인 className에 대해서 질문하셔서 제가 className을 부여하는 방법을 말씀드리자면
우선 nesting으로 확실한 부모 태그가 있다라는 가정하에
전체를 감싸고 있는 태그에는 container라는 이름을 주로 붙입니다.